css-paint-api/Overview.bs

```diff
@@ -244,15 +253,15 @@ called, the user agent <em>must</em> run the following steps:
 
         2. <a for=list>Append</a> |parsedSyntax| to |inputArgumentSyntaxes|.
 
-    12. Let |alphaValue| be the result of <a>Get</a>(|paintCtor|, "alpha").
+    12. Let |alphaValue| be the result of <a>Get</a>(|paintCtor|, "ContextAttributes.alpha").
```

Comment:
This should be...

12. Let |contextAttributesValue| be the result of <a>Get</a>(|paintCtor|, "contextAttributes").

13. Let |contextAttributes| be the result of <a>converting</a> |contextAttributesValue| to a {{PaintContextAttributes}}, .... etc etc etc